### PR TITLE
Added Activities tab and search functionalities

### PR DESCRIPTION
**Pull request by Rohanmond**
Added debouncer in search functionalities
added activities tab in profile

### CODE REVIEW

**Review by rutvikpumak — APPROVED** [review]
Code looks well and functionality working as expected

**Review by znjs — APPROVED** [review]
- Clean Code
- Good PR

File: src/components/Nav/Nav.jsx [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+  }, [searchInput]);
+
+  useEffect(() => {
+    if (resetMenu) {
+      setShowSearch(false);
+      setSearchInput('');
+      setSearchData([]);
+    }
+  }, [resetMenu]);
+  console.log(searchedData, 'search');
```
Comment (Parul1999) [inline src/components/Nav/Nav.jsx]: Remove console

**Review by Parul1999 — APPROVED** [review]
The code structure looks good and can be merged. 
after removing the console logs .

**Pull request merged**